pages/components/Footer.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Image from "next/image";
+import logo from "../../stories/assets/logo.png";
+
+const Footer = () => {
+  return (
+    <footer
+      style={{
+        width: "100%",
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        textAlign: "center",
```

Comment:
Not needed. Please stick to the AC. Also discussed on LinkedIn 👇🏽 

>  just go without position, just leave the width:100% and we are good to go 😊

<img width="823" alt="Conversation on additional styling" src="https://github.com/YurisCodingClub/accessibility-mentor/assets/54622834/e3e17ab5-ace6-456c-a6fa-d9f4ee7cfbd4">

**Next steps** 👉🏽 Remove everything except the width. Only do as much as described in the AC to get your PRs merged. The design of the footer will look nothing alike as it is now. So, all this code is going to be removed with the next ticket for the footer already, which is unnecessary work for the next contributor as well as for the maintainer mentioning this step in the ticket.

**Wish** 🙏🏽 : Please talk and explain reasoning behind your decisions for me to understand certain things better. If there are things you do not understand, ask me! This prevents unnecessary work for both of us which is very much appreciated to create a smooth work experience.

Reply:
Thanks for your input with this @YuriDevAT. I was going back and forth with using Flexbox. I wanted to show of my knowledge a bit more (plus Flexbox is easier to use in my experience), but I wanted to follow the task list. Unfortunately, I went with my slightly more ambitious side. No worries, I'll definitely stick with your suggestions this time.

Reply:
I know the feeling all to well 😁. But you will be able to show off your skills when we dive deeper into the code base, I promise 🤞🏽